documentation/course/e2/README.md

```diff
@@ -2410,13 +2410,42 @@ TODO @mlodybercik: Dodać diagram pakietów, opis architektury i endpointy.
 
 ![Realizacja przypadku użycia - Tworzenie konta](./images/sequence-diagram-jobberknoll-create-account.drawio.svg)
 
-## PU `TIC/??`
+## PU `TIC/01`, `TIC/02`, `TIC/03`, `TIC/04`, `TIC/05`
+
+> Jako _pasażer_ chcę mieć możliwość _zakupu_ _biletu jednorazowego_.
+> Jako _pasażer_ chcę mieć możliwość _zakupu_ _biletu czasowego_.
+> Jako _pasażer_ chcę mieć możliwość _zakupu_ _biletu okresowego_.
+> Jako _pasażer_ chcę mieć możliwość _zakupu_ _biletu ulgowego_.
+> Jako _pasażer_ chcę mieć możliwość _zakupu_ _biletu normalnego_.
```

Comment:
**chore:** Quotes get merged into one and don't respect newlines when written like this, please add a blank line between them (same below).